pkg/controller/controller.go

```diff
@@ -522,3 +523,38 @@ func isInstanceReady(instance *v1alpha1.Instance) bool {
 
 	return false
 }
+
+func IsOwnedBy(obj metav1.Object, owner metav1.Object) bool {
```

Comment:
This method and below could be used for other resource types in future.
For example, for implementing ownership of `Bindings` by `Instance`